CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2025-03-19 - v6.9.3
+### Fixes
+* Addressed racing conditions when showing coupled interactions that are engaged from activities without supportFragmentManager. 
```

Comment:
No biggie but I think "race condition" is the more common phrasing, e.g. "Addressed a race condition when showing..."